app/javascript/components/redfish-server-provision-dialog.jsx

```diff
@@ -85,7 +79,8 @@ const RedfishServerProvisionDialog = ({ dispatch }) => {
       schema={schema}
       onSubmit={submitValues}
       showFormControls={false}
-      onStateUpdate={handleFormStateUpdate}
+      debug={handleFormStateUpdate}
```

Comment:
is this an old version before the formspy?

Reply:
No, if you go a few lines above, this is mapping a field value to the state. Therefore, it's not able to leverage the SpyField :disappointed: I'll fix it in a seaparate PR, so until then I'm using debug. Btw a few lines above there's some comment about it :wink:

Reply:
I've read the comment, still didn't know if it was up to date :) 

OK, works for me :+1:

Reply:
I have a feeling that we can get rid of this with a little rewrite which I'm planning after this is in :wink: